File: src/test/java/com/whale/growthnote/domain/test/CategoryRepositoryTest.java
package com.whale.growthnote.domain.test;

import com.whale.growthnote.domain.category.entity.Category;
import com.whale.growthnote.domain.category.repository.CategoryRepository;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
class CategoryRepositoryTest {

	@Autowired
	private CategoryRepository categoryRepository;

	@Test
	void test() {
		Category category = Category.builder()
			.name("name1")
			.build();
		categoryRepository.save(category);
	}
}

File: src/test/kotlin/com/whale/growthnote/domain/category/repository/CategoryRepositoryTest.kt
package com.whale.growthnote.domain.category.repository

import com.whale.growthnote.GrowthNoteApplication
import io.kotest.core.spec.style.DescribeSpec
import io.kotest.matchers.shouldBe
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.context.annotation.Import


@SpringBootTest
class CategoryRepositoryTest(
    @Autowired val categoryRepository: CategoryRepository
) : DescribeSpec() {

    init {
        describe("describe") {
            it("it") {
                categoryRepository.findContainsName("")
            }
        }
    }
}


